src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "communication email",
+)
```

Comment:
Why the generated folder name is NOT `communication\email`? Please do not manually change the folde of `aaz`, it should be directly generated from `aaz-dev-tools`.